### ISSUE TITLE
Accessibility Specs

### ISSUE THREAD

**Issue by orangewolf** [notch8/ArchivesSpace#79]
Ref #100 
# Summary

2 of the public accessibility specs are failing.

# Acceptance Criteria

- [ ] All spec failures are resolved

# Screenshots or Video
Public:
![image](/uploads/67f895a489830158e4b6fd18c5c677d3/image.png)
Frontend:
![image](/uploads/c5b242a493ef9ca1dfeeb7b1485498b9/image.png)

# Testing Instructions

Start the app: `docker-compose up app`
Bash into the container: `docker-compose exec app bash`
Unset frontend proxy URL: `unset APPCONFIG_FRONTEND_PROXY_URL`
Run test set for accessibility (separate commands for frontend & public):
    `./build/run rspec -Ddir="../public" -Dtag="db:accessibility" -Dspec="features" -Dorder="defined"`
    `./build/run rspec -Ddir="../frontend" -Dtag="db:accessibility" -Dspec="features" -Dorder="defined"`
